core/embed/rust/src/ui/display.rs

```diff
@@ -57,6 +60,27 @@ pub fn icon(center: Point, data: &[u8], fg_color: Color, bg_color: Color) {
     );
 }
 
+// Used on T1 only.
+pub fn rounded_rect1(r: Rect, fg_color: Color, bg_color: Color) {
```

Comment:
I think this is fine as it is for now, but in the future i'd really like to expose a more low-level interface to the display for both models. Something like `draw_iter(r: Rect, it: impl IntoIterator<Item = Color>)`.